### PR TITLE
[#7760] Stage irodsServerMonPerf for msiServerMonPerf test (4-3-stable)

### PR DESCRIPTION
**Pull request by alanking**
In service of #7760 

Companion PR: https://github.com/irods/irods/pull/7830
Companion PR: https://github.com/irods/irods_client_icommands/pull/490

### CODE REVIEW

File: scripts/irods/test/test_all_rules.py [hunk 1]
```diff
@@ -489,6 +486,46 @@ def test_writeLine_config_first_3477(self):
             IrodsController().start()
 
 
+    @unittest.skipUnless(plugin_name == 'irods_rule_engine_plugin-irods_rule_language', 'only applicable for irods_rule_language REP')
+    def test_msiServerMonPerf_default_3736(self):
+        path_to_irodsServerMonPerf = '/'.join([paths.home_directory(), 'msiExecCmd_bin', 'irodsServerMonPerf'])
```
Comment (SwooshyCueb) [inline scripts/irods/test/test_all_rules.py:491]: Consider using [`os.path.join`](https://docs.python.org/3.5/library/os.path.html#os.path.join) instead.

Comment (alanking) [reply scripts/irods/test/test_all_rules.py:491]: Oh, good point. I'd gotten into this habit due to PRC running on Windows and then trying to construct iRODS logical paths. This is referring to a path on the local filesystem, so `os.path.join` should be better here.

Comment (alanking) [reply scripts/irods/test/test_all_rules.py]: Done

File: scripts/irods/test/test_all_rules.py [hunk 2]
```diff
@@ -489,6 +486,46 @@ def test_writeLine_config_first_3477(self):
...
+    @unittest.skipUnless(plugin_name == 'irods_rule_engine_plugin-irods_rule_language', 'only applicable for irods_rule_language REP')
+    def test_msiServerMonPerf_default_3736(self):
+        path_to_irodsServerMonPerf = os.path.join(paths.home_directory(), 'msiExecCmd_bin', 'irodsServerMonPerf')
+        path_to_irodsServerMonPerf_template = path_to_irodsServerMonPerf + '.template'
+        irodsServerMonPerf_already_existed = os.path.exists(path_to_irodsServerMonPerf)
+
+        rule_file="test_msiServerMonPerf.r"
+        rule_string= '''
+msiTestServerMonPerf {{
+    msiServerMonPerf("verbose", "default");
```
Comment (SwooshyCueb) [inline scripts/irods/test/test_all_rules.py:498]: The test rule file in the icommands repo being replaced by this test runs it both ways, so we probably should here too.

```suggestion
    msiServerMonPerf("default", "default");
    msiServerMonPerf("verbose", "default");
```

Comment (alanking) [reply scripts/irods/test/test_all_rules.py:499]: Also a good point. Done.

**Review by korydraughn — APPROVED** [review]
Seems good to me.

**Comment by alanking** [comment]
Added #s. Will merge with https://github.com/irods/irods/pull/7830

**Pull request merged**